src/main/java/com/AdelaQA/classesandobjects/autoservicetask/ManageAutoService.java

```diff
@@ -0,0 +1,30 @@
+package com.AdelaQA.classesandobjects.autoservicetask;
+
+public class ManageAutoService {
+    public static void main(String[] args) {
+       // Declararea si initializarea obiectelor folosind toate tipurile de constructori
+       Car mercedesBenzSeriaE = new Car(makeParametruDeIntrare: "E Class", modelFormalParameter: "Mercedes", colorLocalVariable: "Verde");
```

Comment:
you shouldn't write the names of parameters into constructor body, only the values =>
new Car("E Class", "Mercedes", "Verde")